# Translations template for PROJECT.
# Copyright (C) 2022 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2022-04-22 09:22+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.9.1\n"

#: src/backend/classes/MaarchWebServices.py:39
msgid "HOST_NOT_FOUND"
msgstr ""

#: src/backend/controllers/accounts.py:47 src/backend/models/accounts.py:51
msgid "GET_SUPPLIER_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:60 src/backend/models/accounts.py:70
msgid "GET_ADDRESS_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:80
#: src/backend/controllers/accounts.py:86
msgid "UPDATE_SUPPLIER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:112
msgid "UPDATE_SUPPLIER_POSITIONS_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:137
msgid "UPDATE_SUPPLIER_PAGES_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:163
#: src/backend/controllers/accounts.py:169
#: src/backend/controllers/accounts.py:195
#: src/backend/controllers/accounts.py:201 src/backend/models/accounts.py:108
msgid "UPDATE_ADDRESS_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:228 src/backend/models/accounts.py:124
msgid "CREATE_ADDRESS_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:262
#: src/backend/controllers/accounts.py:268 src/backend/models/accounts.py:140
msgid "CREATE_SUPPLIER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:269
msgid "SUPPLIER_VAT_NUMBER_ALREADY_EXISTS"
msgstr ""

#: src/backend/controllers/accounts.py:292 src/backend/controllers/user.py:99
#: src/backend/models/accounts.py:192 src/backend/models/accounts.py:211
#: src/backend/models/accounts.py:226 src/backend/models/user.py:133
msgid "GET_CUSTOMER_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:334
#: src/backend/controllers/accounts.py:340 src/backend/models/accounts.py:244
msgid "UPDATE_CUSTOMER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:371
#: src/backend/controllers/accounts.py:377 src/backend/models/accounts.py:260
msgid "CREATE_CUSTOMER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:378
msgid "CUSTOMER_VAT_NUMBER_ALREADY_EXISTS"
msgstr ""

#: src/backend/controllers/accounts.py:394
#: src/backend/controllers/accounts.py:400 src/backend/models/accounts.py:278
msgid "DELETE_CUSTOMER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:417
#: src/backend/controllers/accounts.py:423 src/backend/models/accounts.py:158
msgid "DELETE_SUPPLIER_ERROR"
msgstr ""

#: src/backend/controllers/accounts.py:441
msgid "LOAD_SUPPLIER_REFERENCIAL_ERROR"
msgstr ""

#: src/backend/controllers/auth.py:80
msgid "LOGIN_ERROR"
msgstr ""

#: src/backend/controllers/auth.py:95 src/backend/controllers/auth.py:103
#: src/backend/controllers/auth.py:105
msgid "JWT_ERROR"
msgstr ""

#: src/backend/controllers/doctypes.py:131
msgid "DOCTYPESEPARATOR"
msgstr ""

#: src/backend/controllers/doctypes.py:134
msgid "DOCUMENTSEPARATOR"
msgstr ""

#: src/backend/controllers/doctypes.py:137
msgid "BUNDLESEPARATOR"
msgstr ""

#: src/backend/controllers/forms.py:66 src/backend/controllers/forms.py:115
#: src/backend/models/forms.py:52
msgid "GET_FORM_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:98
msgid "GET_FORM_FIELDS_BY_FORM_ID_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:162 src/backend/controllers/forms.py:168
msgid "DELETE_FORM_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:184
#: src/backend/controllers/positions_masks.py:154
msgid "COPY_OF"
msgstr ""

#: src/backend/controllers/forms.py:200 src/backend/controllers/forms.py:206
msgid "DUPLICATE_FORM_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:223 src/backend/controllers/forms.py:229
msgid "DISABLE_FORM_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:246 src/backend/controllers/forms.py:252
msgid "ENABLE_FORM_ERROR"
msgstr ""

#: src/backend/controllers/forms.py:285 src/backend/controllers/forms.py:291
msgid "UPDATE_FORMS_FIELDS_ERROR"
msgstr ""

#: src/backend/controllers/history.py:28
msgid "ADD_HISTORY_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:49 src/backend/controllers/inputs.py:55
msgid "UPDATE_INPUT_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:82 src/backend/controllers/inputs.py:93
#: src/backend/controllers/inputs.py:106
msgid "CREATE_INPUT_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:83
msgid "INPUT_ID_ALREADY_EXISTS"
msgstr ""

#: src/backend/controllers/inputs.py:94
msgid "INPUT_FOLDER_ALREADY_EXISTS"
msgstr ""

#: src/backend/controllers/inputs.py:119
msgid "GET_INPUT_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:143 src/backend/controllers/inputs.py:149
msgid "DELETE_INPUT_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:178
msgid "FS_WATCHER_DELETION_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:179 src/backend/controllers/inputs.py:236
msgid "FS_WATCHER_CONFIG_DOESNT_EXIST"
msgstr ""

#: src/backend/controllers/inputs.py:217 src/backend/controllers/inputs.py:235
msgid "FS_WATCHER_CREATION_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:218
msgid "CAN_NOT_CREATE_FOLDER_PERMISSION_ERROR"
msgstr ""

#: src/backend/controllers/inputs.py:241
msgid "SCRIPT_SAMPLE_DOESNT_EXISTS"
msgstr ""

#: src/backend/controllers/inputs.py:247
msgid "SCRIPT_FOLDER_DOESNT_EXISTS"
msgstr ""

#: src/backend/controllers/outputs.py:72 src/backend/controllers/outputs.py:78
msgid "UPDATE_OUTPUT_ERROR"
msgstr ""

#: src/backend/controllers/outputs.py:104
msgid "CREATE_OUTPUT_ERROR"
msgstr ""

#: src/backend/controllers/outputs.py:117
msgid "GET_OUTPUT_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/outputs.py:130
msgid "GET_OUTPUT_TYPE_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/outputs.py:147
#: src/backend/controllers/outputs.py:153
msgid "DELETE_OUTPUT_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:75
#: src/backend/controllers/positions_masks.py:92
msgid "GET_POSITION_MASK_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:137
#: src/backend/controllers/positions_masks.py:143
msgid "DELETE_POSITION_MASK_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:170
#: src/backend/controllers/positions_masks.py:176
msgid "DUPLICATE_POSITIONS_MASKS_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:193
#: src/backend/controllers/positions_masks.py:199
msgid "DISABLE_POSITION_MASK_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:216
#: src/backend/controllers/positions_masks.py:222
msgid "ENABLE_POSITION_MASK_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:247
msgid "UPDATE_POSITIONS_BY_POSITIONS_MASK_ID_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:272
msgid "UPDATE_PAGES_BY_POSITIONS_MASK_ID_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:292
msgid "DELETE_POSITIONS_BY_POSITION_MASK_ID_ERROR"
msgstr ""

#: src/backend/controllers/positions_masks.py:312
msgid "DELETE_PAGES_BY_POSITION_MASK_ID_ERROR"
msgstr ""

#: src/backend/controllers/privileges.py:36
#: src/backend/controllers/privileges.py:64
msgid "GET_PRIVILEGES_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:53 src/backend/controllers/roles.py:59
#: src/backend/controllers/roles.py:106 src/backend/controllers/roles.py:112
#: src/backend/models/roles.py:70
msgid "UPDATE_ROLE_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:84 src/backend/models/roles.py:86
msgid "CREATE_ROLE_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:125
msgid "GET_ROLE_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:142 src/backend/controllers/roles.py:148
msgid "DELETE_ROLE_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:165 src/backend/controllers/roles.py:171
msgid "DISABLE_ROLE_ERROR"
msgstr ""

#: src/backend/controllers/roles.py:188 src/backend/controllers/roles.py:194
msgid "ENABLE_ROLE_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:79
msgid "LOAD_METADATA_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:107
#: src/backend/controllers/verifier.py:77
msgid "AT"
msgstr ""

#: src/backend/controllers/splitter.py:224
msgid "ADD_DOCUMENT_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:275
#: src/backend/controllers/splitter.py:281
#: src/backend/controllers/verifier.py:441
#: src/backend/controllers/verifier.py:447
#: src/backend/controllers/verifier.py:453
msgid "EXPORT_MAARCH_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:287
#: src/backend/controllers/verifier.py:465
msgid "MAARCH_WS_INFO_EMPTY"
msgstr ""

#: src/backend/controllers/splitter.py:313
msgid "EXPORT_PDF_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:330
#: src/backend/controllers/splitter.py:550
#: src/backend/controllers/splitter.py:569
#: src/backend/controllers/verifier.py:557
msgid "EXPORT_XML_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:351
msgid "UPDATE_BATCH_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:366
msgid "UPDATE_DOCUMENTS_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:382
msgid "UPDATE_DOCUMENT_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:397
#: src/backend/controllers/splitter.py:418
#: src/backend/controllers/splitter.py:433
#: src/backend/controllers/splitter.py:448
msgid "UPDATE_PAGES_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:462
msgid "CMIS_CONNECTION_ERROR"
msgstr ""

#: src/backend/controllers/splitter.py:640
#: src/backend/controllers/verifier.py:706 src/backend/models/splitter.py:401
#: src/backend/models/verifier.py:137
msgid "GET_TOTALS_ERROR"
msgstr ""

#: src/backend/controllers/status.py:35
msgid "RETRIEVES_STATUS_ERROR"
msgstr ""

#: src/backend/controllers/user.py:31
msgid "CREATE_USER_ERROR"
msgstr ""

#: src/backend/controllers/user.py:69 src/backend/models/user.py:114
msgid "GET_USER_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/user.py:113
msgid "UPDATE_PROFILE"
msgstr ""

#: src/backend/controllers/user.py:114
msgid "ERROR_OLD_PASSWORD_NOT_MATCH"
msgstr ""

#: src/backend/controllers/user.py:119
msgid "UPDATE_USER"
msgstr ""

#: src/backend/controllers/user.py:120
msgid "ERROR_PASSWORD_NOT_MATCH"
msgstr ""

#: src/backend/controllers/user.py:147 src/backend/controllers/user.py:153
#: src/backend/models/user.py:152
msgid "UPDATE_USER_ERROR"
msgstr ""

#: src/backend/controllers/user.py:170 src/backend/controllers/user.py:176
msgid "DELETE_USER_ERROR"
msgstr ""

#: src/backend/controllers/user.py:193 src/backend/controllers/user.py:199
msgid "DISABLE_USER_ERROR"
msgstr ""

#: src/backend/controllers/user.py:216 src/backend/controllers/user.py:222
msgid "ENABLE_USER_ERROR"
msgstr ""

#: src/backend/controllers/user.py:242 src/backend/controllers/user.py:248
#: src/backend/models/user.py:170
msgid "UPDATE_CUSTOMERS_USER_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:58 src/backend/models/verifier.py:34
msgid "GET_INVOICE_BY_ID_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:165
#: src/backend/controllers/verifier.py:255
msgid "UPDATE_INVOICE_POSITIONS_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:190
#: src/backend/controllers/verifier.py:275
msgid "UPDATE_INVOICE_PAGES_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:214
#: src/backend/controllers/verifier.py:234
msgid "UPDATE_INVOICE_DATA_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:292
#: src/backend/controllers/verifier.py:298
msgid "DELETE_INVOICE_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:316
#: src/backend/controllers/verifier.py:322
msgid "UPDATE_INVOICE_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:342
msgid "REMOVE_LOCK_BY_USER_ID_ERROR"
msgstr ""

#: src/backend/controllers/verifier.py:448
msgid "PDF_FILE_NOT_FOUND"
msgstr ""

#: src/backend/controllers/verifier.py:459
msgid "MAARCH_WS_INFO_WRONG"
msgstr ""

#: src/backend/controllers/verifier.py:551
msgid "XML_DESTINATION_FOLDER_DOESNT_EXISTS"
msgstr ""

#: src/backend/controllers/verifier.py:633
#: src/backend/controllers/verifier.py:636
#: src/backend/controllers/verifier.py:649
#: src/backend/controllers/verifier.py:666
msgid "API_INSEE_ERROR_CONNEXION"
msgstr ""

#: src/backend/controllers/verifier.py:688
msgid "VAT_NOT_VALID"
msgstr ""

#: src/backend/controllers/verifier.py:692
msgid "VAT_API_ERROR"
msgstr ""

#: src/backend/models/accounts.py:90
msgid "SUPPLIER_UPDATE_ERROR"
msgstr ""

#: src/backend/models/auth.py:35 src/backend/models/user.py:37
msgid "BAD_USERNAME"
msgstr ""

#: src/backend/models/auth.py:37 src/backend/models/user.py:39
msgid "BAD_PASSWORD"
msgstr ""

#: src/backend/models/auth.py:39
msgid "USER_DELETED"
msgstr ""

#: src/backend/models/auth.py:41
msgid "USER_DISABLED"
msgstr ""

#: src/backend/models/config.py:131
msgid "UPDATE_CONFIGURATION_ERROR"
msgstr ""

#: src/backend/models/config.py:149
msgid "UPDATE_REGEX_ERROR"
msgstr ""

#: src/backend/models/config.py:171
msgid "UPDATE_DOCSERVER_ERROR"
msgstr ""

#: src/backend/models/custom_fields.py:45
msgid "ADD_CUSTOM_FIELD_ERROR"
msgstr ""

#: src/backend/models/custom_fields.py:48
msgid "CUSTOM_FIELD_ALREADY_EXIST"
msgstr ""

#: src/backend/models/custom_fields.py:87
msgid "UPDATE_CUSTOM_FIELDS_ERROR"
msgstr ""

#: src/backend/models/custom_fields.py:107
msgid "DELETE_CUSTOM_FIELDS_ERROR"
msgstr ""

#: src/backend/models/doctypes.py:45
msgid "ADD_DOCTYPE_ERROR"
msgstr ""

#: src/backend/models/doctypes.py:48
msgid "DOCTYPE_ALREADY_EXIST"
msgstr ""

#: src/backend/models/doctypes.py:85
msgid "UPDATE_DOCTYPE_ERROR"
msgstr ""

#: src/backend/models/doctypes.py:112 src/backend/models/doctypes.py:115
msgid "SET_DEFAULT_DOCTYPE_ERROR"
msgstr ""

#: src/backend/models/forms.py:71
msgid "GET_DEFAULT_FORM_ERROR"
msgstr ""

#: src/backend/models/forms.py:91
msgid "UPDATE_FORM_ERROR"
msgstr ""

#: src/backend/models/forms.py:109
msgid "UPDATE_FORM_FIELDS_ERROR"
msgstr ""

#: src/backend/models/forms.py:157
msgid "ADD_FORM_ERROR"
msgstr ""

#: src/backend/models/forms.py:160
msgid "FORM_LABEL_ALREADY_EXIST"
msgstr ""

#: src/backend/models/inputs.py:50
msgid "INPUT_DOESNT_EXISTS"
msgstr ""

#: src/backend/models/inputs.py:84
msgid "INPUT_UPDATE_ERROR"
msgstr ""

#: src/backend/models/inputs.py:100
msgid "INPUT_CREATE_ERROR"
msgstr ""

#: src/backend/models/outputs.py:64
msgid "OUTPUT_DOESNT_EXISTS"
msgstr ""

#: src/backend/models/outputs.py:83
msgid "OUTPUT_TYPE_DOESNT_EXISTS"
msgstr ""

#: src/backend/models/outputs.py:103
msgid "OUTPUT_UPDATE_ERROR"
msgstr ""

#: src/backend/models/outputs.py:119
msgid "OUTPUT_CREATE_ERROR"
msgstr ""

#: src/backend/models/positions_masks.py:52
msgid "GET_POSITIONS_MASKS_BY_ID_ERROR"
msgstr ""

#: src/backend/models/positions_masks.py:72
msgid "UPDATE_POSITIONS_MASKS_ERROR"
msgstr ""

#: src/backend/models/positions_masks.py:90
msgid "UPDATE_POSITIONS_MASKS_FIELDS_ERROR"
msgstr ""

#: src/backend/models/positions_masks.py:120
msgid "ADD_POSITIONS_MASKS_ERROR"
msgstr ""

#: src/backend/models/positions_masks.py:123
msgid "POSITIONS_MASK_LABEL_ALREADY_EXIST"
msgstr ""

#: src/backend/models/privileges.py:32 src/backend/models/privileges.py:49
msgid "ERROR_RETRIEVING_PRIVILEGES"
msgstr ""

#: src/backend/models/roles.py:50
msgid "ROLE_DOESNT_EXISTS"
msgstr ""

#: src/backend/models/roles.py:104
msgid "ROLE_PRIVILEGES_UPDATE_ERROR"
msgstr ""

#: src/backend/models/roles.py:122
msgid "ROLE_PRIVILEGES_CREATE_ERROR"
msgstr ""

#: src/backend/models/splitter.py:64
msgid "GET_MAX_SPLIT_INDEX_ERROR"
msgstr ""

#: src/backend/models/splitter.py:103
msgid "UPDATE_SETTINGS_ERROR"
msgstr ""

#: src/backend/models/splitter.py:124
msgid "INSERT_PAGE_ERROR"
msgstr ""

#: src/backend/models/splitter.py:182
msgid "GET_BATCH_ERROR"
msgstr ""

#: src/backend/models/splitter.py:201
msgid "GET_DOCUMENTS_ERROR"
msgstr ""

#: src/backend/models/splitter.py:220
msgid "GET_PAGES_ERROR"
msgstr ""

#: src/backend/models/splitter.py:238
msgid "GET_MAX_SOURCE_PAGE_VALUE_ERROR"
msgstr ""

#: src/backend/models/splitter.py:257
msgid "GET_DOCUMENT_ERROR"
msgstr ""

#: src/backend/models/splitter.py:275
msgid "GET_DOCUMENT_MAX_SPLIT_INDEX_ERROR"
msgstr ""

#: src/backend/models/status.py:34
msgid "NO_STATUS"
msgstr ""

#: src/backend/models/user.py:41
msgid "USER"
msgstr ""

#: src/backend/models/user.py:41
msgid "ALREADY_REGISTERED"
msgstr ""

#: src/backend/models/verifier.py:86 src/backend/models/verifier.py:103
msgid "INVOICE_UPDATE_ERROR"
msgstr ""

#: src/backend/rest/auth.py:36
msgid "PGSQL_ERROR"
msgstr ""

#: src/backend/rest/splitter.py:39 src/backend/rest/verifier.py:37
msgid "UNKNOW_ERROR"
msgstr ""

